tsconfig.json

```diff
@@ -14,6 +14,7 @@
     ]
   },
   "exclude": [
-    "node_modules"
+    "node_modules",
+    "dist"
```

Comment:
I had to add this in order to stop VS Code from throwing an error - not sure if it should be there or not.